pages/design/rule-design.md

```diff
@@ -39,34 +63,56 @@ A test aspect is a part of a test subject that must be available in order to pro
 
 Other aspects may be necessary for testing. These can be added as long as they are sufficiently defined.
 
+For more details, see [ACT Rules Format: Input](https://www.w3.org/TR/act-rules-format/#input)
+
 ### Authors
 
 Names of the Authors. These must be an exact match of names in `contributors` property in `package.json`.
 
+## Applicability
```

Comment:
Both for Applicability and Expectation: recall that they must be correct English sentences (final full period, not skipping article, …)
For example, I think that "Target element has an accessible name" should be "The targe element has an accessible name." (or "The test target has an accessible name.")

Reply:
OK